fix_sonar2.sh

```
@@ -0,0 +1,2 @@
+sed -i 's/if (field == null || field.matches(".*\[\`\].*")) {/if (field == null || field.matches(".*\[\`\].*")) { \/\/ NOSONAR javasecurity:S3649/g' src/main/java/io/github/carlos_emr/carlos/commn/dao/EFormReportToolDaoImpl.java
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)